<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculator1</class>
 <widget class="QWidget" name="calculator1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>calculator1</string>
  </property>
  <widget class="QLineEdit" name="ans">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>-30</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>351</width>
     <height>101</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="ans1">
   <property name="geometry">
    <rect>
     <x>422</x>
     <y>-30</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>113</width>
     <height>101</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>141</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="less">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>61</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>81</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>81</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>71</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_9">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>61</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>71</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_10">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>420</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>81</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_11">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>81</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_12">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>111</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="NUMBER">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_13">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>71</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_14">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>71</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-230</x>
     <y>-30</y>
     <width>1041</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/C:/Users/11861/Desktop/530e0bfe97150.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>ans</zorder>
  <zorder>ans1</zorder>
  <zorder>confirm</zorder>
  <zorder>less</zorder>
  <zorder>confirm_3</zorder>
  <zorder>confirm_4</zorder>
  <zorder>confirm_5</zorder>
  <zorder>confirm_6</zorder>
  <zorder>confirm_7</zorder>
  <zorder>confirm_8</zorder>
  <zorder>confirm_9</zorder>
  <zorder>confirm_10</zorder>
  <zorder>confirm_11</zorder>
  <zorder>confirm_12</zorder>
  <zorder>NUMBER</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>confirm_13</zorder>
  <zorder>confirm_14</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
